iota/types.py

```diff
@@ -60,6 +60,12 @@ def random(cls, length):
     alphabet  = list(itervalues(AsciiTrytesCodec.alphabet))
     generator = SystemRandom()
 
+    try:
+      length = length or cls.LEN
```

Comment:
This would also trip if `length == 0`; could we change this to an explicit `is None` check?